test/gurka/gurka.h

```diff
@@ -593,18 +598,6 @@ findEdge(valhalla::baldr::GraphReader& reader,
   return std::make_tuple(baldr::GraphId{}, nullptr, baldr::GraphId{}, nullptr);
 }
 
-std::tuple<const baldr::GraphId,
-           const baldr::DirectedEdge*,
-           const baldr::GraphId,
-           const baldr::DirectedEdge*>
-findEdge(const map& map,
-         const baldr::GraphId& tile_id,
-         const std::string& way_name,
-         const std::string& end_node) {
-  baldr::GraphReader reader(map.config.get_child("mjolnir"));
-  return findEdge(reader, map.nodes, tile_id, way_name, end_node);
```

Comment:
we dont want to do this because we return pointers into data that will no longer exist once the graphreader goes out of scope at the end of this function